pkg/simple/client/devops/jenkins/options.go

```diff
@@ -25,10 +25,14 @@ import (
 )
 
 type Options struct {
-	Host           string `json:",omitempty" yaml:"host" description:"Jenkins service host address"`
-	Username       string `json:",omitempty" yaml:"username" description:"Jenkins admin username"`
-	Password       string `json:",omitempty" yaml:"password" description:"Jenkins admin password"`
-	MaxConnections int    `json:"maxConnections,omitempty" yaml:"maxConnections" description:"Maximum connections allowed to connect to Jenkins"`
+	Host                       string `json:",omitempty" yaml:"host" description:"Jenkins service host address"`
+	Username                   string `json:",omitempty" yaml:"username" description:"Jenkins admin username"`
+	Password                   string `json:",omitempty" yaml:"password" description:"Jenkins admin password"`
+	MaxConnections             int    `json:"maxConnections,omitempty" yaml:"maxConnections" description:"Maximum connections allowed to connect to Jenkins"`
+	DevOpsServiceAddress       string `json:"devopsServiceAddress" yaml:"devopsServiceAddress" description:"The separate DevOps Service address"`
+	DevOpsPluginServiceAddress string `json:"devopsPluginServiceAddress" yaml:"devopsPluginServiceAddress" description:"The KubeSphere DevOps plugin Service address"`
+	// K8sBearerToken does not take input from outside
+	K8sBearerToken string
```

Comment:
Why do we need `K8sBearerToken`?

Reply:
The purpose of the variable `K8sBearerToken`. https://github.com/kubesphere/kubesphere/pull/4106/files#diff-1f0f138b59d0cd61e116c34a0f0fdda8c23c48cd517d1f0c9946ea6d4d1d3930R112

This also is a temporary solution to pass the token.